x-pack/plugins/apm/public/components/app/profiling_overview/index.tsx

```diff
@@ -67,6 +74,8 @@ export function ProfilingOverview() {
               end={end}
               environment={environment}
               dataSource={preferred?.source}
+              searchBarFilter={searchBarFilter}
+              onSearchBarFilterChange={setSearchBarFilter}
```

Comment:
On change, It appears that the URL is not updated, and the query does not persist when switching APM tabs. is this on purpose?

Reply:
That was on purpose, but I talked to the PM and I'm adding it to the URL, thanks for that.